addons/guide_08_recommended_servers/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "api_version": "0.1",
+  "id": "guide_08_recommended_servers",
+  "name": "Guide: recommended servers",
+  "type": "guide",
```

Comment:
if we are not 100% sure about the strings, let's add `translatable: false`

Reply:
Added. I think this is meant to be final content but we don't need this guide immediately so I've added `translatable: false` for now and am following up with Santiago to verify.